daemon/logger/awslogs/cloudwatchlogs.go

```diff
@@ -309,48 +371,87 @@ var newTicker = func(freq time.Duration) *time.Ticker {
 func (l *logStream) collectBatch() {
```

Comment:
@Can you update the comment for `collectBatch` to include the new multiline behavior and the fact that some of the logic has been moved to `processEvents`?